kgo.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const foreign = require('foreign');
+const kgo = require('kgo');
+const R = require('ramda');
+const S = require('sanctuary');
+const shuv = require('shuv');
```

Comment:
I'd prefer to use [`R.partial`][1] rather than introduce an extra dependency.


[1]: http://ramdajs.com/0.18.0/docs/#partial